Jenkinsfile

```diff
@@ -60,6 +60,7 @@ pipeline {
                 expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME == 'develop' }
```

Comment:
Add `beforeAgent true` to the when block. That way condition is evaluated before agent is started. Saves build time

Reply:
Done